### ISSUE TITLE
cdr-rating a11y text is not correct

### ISSUE THREAD

**Issue by jnields** [rei/rei-cedar-vue-2#408]
rating of 4.81849 stars should have a screen-reader text of 'Four and three-quarter stars' as it's what is represented visually. instead it shows 4.81849 stars

**Comment by lancetharper** [comment]
What version of rating? SR text says 4.75 for me on v0.1.0
